src/plone/restapi/services/system/configure.zcml

```diff
@@ -0,0 +1,13 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone">
+
+  <plone:service
+    method="GET"
+    for="zope.interface.Interface"
```

Comment:
Wasn't it IRequest where we usually bound to?

Reply:
Fixed